staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/mutating/dispatcher.go

```diff
@@ -202,6 +201,10 @@ func (a *mutatingDispatcher) Dispatch(ctx context.Context, attr admission.Attrib
 		}
 
 		if callErr, ok := err.(*webhookutil.ErrCallingWebhook); ok {
+			if errors.Is(callErr.Reason, context.Canceled) {
+				klog.Warningf("Context Canceled when calling webhook %v", hook.Name)
+				return apierrors.NewInternalError(err)
```

Comment:
same here, not necessarily the same behavior. The blocks for `ignoreClientCallFailures` and outside that block handle the error differently

Reply:
utilruntime.HandleError(callErr) will returns "failed calling webhook "eks-aws-auth-configmap-validation-webhook.amazonaws.com": failed to call webhook: Post "https://127.0.0.1:21375/mutate?timeout=2s": context canceled" so keeping this one.

Reply:
So @liggitt I think if context is cancelled it makes sense to just return this error. Previous behavior was it would `continue` `if ignoreClientCallFailures {` but we can probably just return and bail out